src/IB/IBFEMethod.cpp

```diff
@@ -3662,6 +3662,12 @@ IBFEMethod::getFromInput(Pointer<Database> db, bool /*is_from_restart*/)
         d_default_interp_spec.use_nodal_quadrature = db->getBool("interp_use_nodal_quadrature");
     else if (db->isBool("IB_use_nodal_quadrature"))
         d_default_interp_spec.use_nodal_quadrature = db->getBool("IB_use_nodal_quadrature");
+    if (d_default_interp_spec.use_nodal_quadrature)
```

Comment:
Probably we should use `QSIMPSON` for second-order meshes, but currently there is only a single `default_interp_spec` and a single `default_spread_spec`.

Reply:
But if we don't specify _anything_ here, then bad things happen in determining the bounding boxes.